src/datasets/batch_view/clients_daily/reference.md

```diff
@@ -258,5 +258,5 @@ root
 # Code Reference
 
 This dataset is generated by
-[`telemetry-batch-view`](https://github.com/mozilla/telemetry-batch-view/tree/master/src/main/scala/com/mozilla/telemetry/views/ClientsDailyView.scala).
+[`telemetry-batch-view`](https://github.com/mozilla/telemetry-batch-view/blob/master/GRAVEYARD.md#main-summary-clients-daily-and-addons).
```

Comment:
This dataset is generated by bigquery-etl now, isn't it?

Reply:
Yep, will update.